packages/react-docsite-components/src/index.demo.tsx

```diff
@@ -8,7 +8,9 @@ createApp(
     examplesOf('Example Group')
       .add('Example 1', () => (
         // eslint-disable-next-line import/no-webpack-loader-syntax
-        <Markdown>{require('!raw-loader!@fluentui/react-docsite-components/src/demo/demo.md') as string}</Markdown>
+        <Markdown>
+          {require('!raw-loader?esModule=false!@fluentui/react-docsite-components/src/demo/demo.md') as string}
```

Comment:
If I'm understanding right, the "as string" is no longer necessary.

Also you should get rid of the whole-project import/no-webpack-loader-syntax disable by fixing line 10 of this file (could just make it a full file disable, it doesn't matter for this file)